php/WP_CLI/Formatter.php

```
@@ -635,7 +651,7 @@ private function show_multiple_fields( $data, $format, $ascii_pre_colorized = fa
 			}
 		}
 
-		foreach ( $data as $key => $value ) {
+		foreach ( (array) $data as $key => $value ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)